spec/index.bs

```diff
@@ -14846,9 +14849,8 @@ and {{GPUTextureUsage/STORAGE_BINDING|GPUTextureUsage.STORAGE_BINDING}} usage re
         <td><!-- no multisampling without RENDER_ATTACHMENT (gpuweb/gpuweb#2465) -->
         <td>
         <td><!-- Vulkan -->
-        <td>2
-        <td>&ndash; <!-- no render target -->
-        <td>&ndash; <!-- no render target -->
+        <td colspan=1>2
```

Comment:
Any reason for the `colspan=1` here and elsewhere?

Reply:
The colspan attribute triggers a CSS rule we defined to add dotted lines on either side of the cell. This makes the dotted lines consistent through the columns. Though I'm considering just adding vertical lines in all tables all the time because this is getting kind of silly.